src/teleop/gui/src/components/AutonTask.vue

```diff
@@ -153,27 +169,31 @@ export default {
     watch: {
         // Publish auton LED color to ESW
         nav_state_color: function (color) {
-            let ledMsg = {
-                type: 'AutonLed',
-                color: 'Null'
-            }
             if (color == navBlue) {
-                ledMsg.color = 'Blue'
-                this.greenHook = false
+                this.ledColor = 'blue'
             } else if (color == navRed) {
-                ledMsg.color = 'Red'
-                this.greenHook = false
-            } else if (color == navGreen && !this.greenHook) {
-                ledMsg.color = 'Green'
-                this.greenHook = true //Accounting for the blinking between navGrey and navGreen
-            }
-            if (!this.greenHook || ledMsg.color == 'Green') {
-                // TODO: Implement this once ESW has this interface back up
-                // this.publish('/auton_led',ledMsg)
+                this.ledColor = 'red'
+            } else if (color == navGreen) {
+                this.ledColor = 'green'
+            } else if (color == navGrey) {
+                this.ledColor = 'off'
             }
         },
     },
 
+    methods: {
+      sendColor() {
+
+        let request = new ROSLIB.ServiceRequest({
+          color: this.ledColor
+        });
+        
+        this.auton_led_client.callService(request, (result) => {
+            // Not sure if we need to do anything w/ success bool
```

Comment:
If it's possible for result.success to be false then we should probably call the service again... but only if we aren't calling it in the 10 Hz interval